scripts/abci.js

```diff
@@ -1,12 +1,21 @@
 require('dotenv-expand')(require('dotenv-safe').config());
 
+const { MongoClient } = require('mongodb');
 const createServer = require('abci');
 
 const createDIContainer = require('../lib/createDIContainer');
 
 const errorHandler = require('../lib/errorHandler');
 
+const waitReplicaSetInitializeFactory = require(
+  '../lib/mongoDb/waitReplicaSetInitializeFactory',
+);
+
 (async function main() {
+  const waitReplicaSetInitialize = waitReplicaSetInitializeFactory(MongoClient);
```

Comment:
Let's use the container in order to resolve function and pass `connectToMongoDB` function as dependency for `waitReplicaSetInitializeFactory`.